docs/transports.md

```diff
@@ -238,7 +257,7 @@ const logger = pino({
     pipeline: [{
       target: './my-transform.js'
     }, {
-      // Use target: 'pino/file' to write to stdout
+      // Use target: 'pino/file' with 'options: { destination: 1 }' to write to STDOUT
```

Comment:
The added fragment to this comment does not seem to correspond to the example. What is it trying to explain?

Reply:
It clarifies the comment that was already there. It states that in case you want to log as-is, instead of prettifying, one needs to use pino/file. Original comment made it unclear how to do so.

Reply:
I'd rather see this changed to have the second transport be the `pino/file` transport. The less we push `pino-pretty` in the docs, the better.

Reply:
done